packages/dds/tree/src/simple-tree/tree.ts

```diff
@@ -108,6 +162,14 @@ export interface TreeViewEvents {
 	 * A batch of changes has finished processing and the view has been updated.
 	 */
 	afterBatch(): void;
+
+	/**
+	 * {@link TreeView.root} has changed.
+	 * This includes going into or out of an `error` state where the root is unavailable due to stored schema changes.
+	 * It also included changes the field which contains the root, meaning setting or clearing an optional root, or changing which node is the root.
```

Comment:
```suggestion
	 * It also includes changes to the field containing the root such as setting or clearing an optional root or changing which node is the root.
```

this wording is still a bit awkward but I think a small improvement over PR. feel free to tweak further.